Tests/scripts/install_content_and_test_integrations.sh

```diff
@@ -23,6 +23,7 @@ fi
 CONF_PATH="./Tests/conf.json"
 
 [ -n "${NIGHTLY}" ] && IS_NIGHTLY=true || IS_NIGHTLY=false
+[ -n "${DEMISTO_SDK_NIGHTLY}" ] && DEMISTO_SDK_NIGHTLY=true || DEMISTO_SDK_NIGHTLY=false
```

Comment:
what's this change?
can you add a comment here?

Reply:
If the DEMISTO_SDK_NIGHTLY variable is set, we set it to true, else, we set it to false.
This way we will always have a value

Reply:
It may be confusing to devs in the future, I think code accessing this env var should consider it optional (may be missing) and use a default, instead.